urmila/HTML/Assignment4/Q3.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        th, td{
+            border-top: 1px solid black;
+            border-left: 1px solid black;
+            border-bottom: 1px solid gainsboro;
+            border-right: 1px solid gainsboro;
+            padding: 5px;
+            border-width:2px;
+        }
+        .a {
+                border-top: 1px solid black;
+                border-left: 1px solid black;
+                border-bottom: 1px solid black;
+        }
+        .b{
+            background-color: cornsilk;
+        }
+        .c{
+            background-color: rgb(140, 140, 236);
+        }
+        .d{
+            background-color: cyan;
+        }
+    </style>
+</head>
+<body>
+    <table style="width: 50%;" border="2" cellpadding="23" >
+        <body>
+              <tr>
+                <th rowspan="3">DAY</th>
+                <th colspan="5">seminar</th>
+            </tr>
+            <tr>
+                <th colspan="2">schedule</th>
+                <th colspan="3"; rowspan="2">Topic</th>
+            </tr>
+            <tr>
+                <th>Begin</th>
+                <th>End</th>
+            </tr>
+            <tr>
+                <td rowspan="2">Monday</td>
+                <td rowspan="2"; class="b">8:00 a.m.</td>
+                <td rowspan="2"; class="c">5:00 p.m.</td>
+                <td>Introduction to XML</td>
+            </tr>
+            <tr>
+                <td>Validity:DTD and Relax NG</td>
+            </tr>
+            <tr>
+                <td rowspan="3">Tuesday</td>
+                <td class="b">8:00 a.m.</td>
+                <td class="b">11:00 a.m.</td>
+                <td rowspan="3"> <div class="a"><div>XPath</div></div> <div class="a"><div>XSL Transformation</div></div> </td>
+            </tr>
+            <tr>
+                <td class="b">11:00 a.m.</td>
+                <td class="d">2:00 p.m.</td>
+            </tr>
+            <tr>
+                <td class="d">2:00 p.m.</td>
+                <td class="c">5:00 p.m.</td>
+            </tr>
+            <tr>
+                <td>Wednesday</td>
+                <td class="b">8:00 a.m.</td>
+                <td class="d">12:00 p.m.</td>
+                <td>XSL Formatting Object</td>
+            </tr>
+        </body>
+    </table>
+</body>
+</html> 
```

Comment:
give proper class names 
make indentation for head and body tag
add new line at the end of file